docs/syncer.md

```diff
@@ -86,4 +86,41 @@ kcp-syncer   1/1     1            1           13m
 
 1. Wait for the kcp sync target to go ready.
 
-TODO(marun)
+```sh
+$ kubectl get synctarget -o jsonpath='{.items[0].status.conditions}' | jq .
+[
+  {
+    "lastTransitionTime": "2022-08-15T14:54:35Z",
+    "status": "True",
+    "type": "Ready"
+  },
+  {
+    "lastTransitionTime": "2022-08-15T14:54:35Z",
+    "status": "True",
+    "type": "HeartbeatHealthy"
+  }
+]
+```
+
+1. Create a deployment
+
+```sh
+$ kubectl create deployment kuard --image gcr.io/kuar-demo/kuard-amd64:blue
+```
+
+Note: replace "gcr.io/kuar-demo/kuard-amd64:blue" with "gcr.io/kuar-demo/kuard-arm64:blue" in case you're running
+an Apple M1 based virtual machine.
```

Comment:
why blue? Is that mentioned somewhere above? Seeing only `<your tag>`.

Reply:
`gcr.io/kuar-demo/kuard-[arm64/amd64]:blue` is an example app image, not the syncer image, no clue why they used the `blue` tag :)

Reply:
see https://github.com/kubernetes-up-and-running/kuard#versions